extensions/amp-consent/0.1/amp-consent.js

```diff
@@ -103,6 +222,23 @@ export class AmpConsent extends AMP.BaseElement {
         // TODO: Handle errors
       });
     }
+    this.notificationUiManager_.onQueueEmpty(() => {
+      if (!this.revokeUI_) {
+        return;
+      }
+      this.element.classList.add('amp-active');
+      this.element.classList.remove('amp-hidden');
+      setStyle(this.revokeUI_, 'display', 'block');
```

Comment:
should the revoke UI be dismissed after click?

Reply:
It should : ) Please ignore all revoke related logic in this PR : )